src/nxdoc/web-ui/web-ui-upgrade-notes/web-ui-upgrade-notes-lts-2021.md

```diff
@@ -126,6 +126,38 @@ Since [ELEMENTS-1124](https://jira.nuxeo.com/browse/ELEMENTS-1124) and [NXP-2869
 
 In LTS 2019, this change is already [available as an option](https://jira.nuxeo.com/browse/WEBUI-291). Feel free to activate it to test the editor and to allow your users to get accustomed to it before migrating.
 
+### Upgrade WebdriverIO v7
+
+The new version of Web UI introduces an upgrade on Webdriver IO, to its latest version, v7. With this upgrade there are a few things to consider when migrating:
+
+#### Delete `babel-preset-env`
+Projects relying on `babel-preset-env`, usually via a local `.babelrc` file, can now be deleted
+
+#### Cucumber v7 update
+With the latest Cucumber version, we need to make sure our imports are also updated from:
+```
+import { Given, When, Then } from 'cucumber';
+```
+to:
+```
+import { Given, When, Then } from '@cucumber/cucumber';
+```
+
+#### The new `findElements`
+Webdriver IO 7 introduces the `findElements` utility, to search for multiple elements on the page, starting from the root. To access the results of `findElements`, and to keep compatibility with older versions, we need to change:
+
+```
+const res = elements(...).value;
+```
+
+to:
+```
+const res = elements(...);
+```
+
+#### Node.js supported version range
+Our Functional Test Framework is now compatible with Node.js versions between `10.23.00` and `15.0.0`.
```

Comment:
`.00`?
Also, not sure if `15.0.0` is included, at least not according to the [package.json](https://github.com/nuxeo/nuxeo-web-ui/blob/maintenance-3.0.x/packages/nuxeo-web-ui-ftest/package.json#L14) file.